gh-pages-stub/public/js/analytics.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014-2017 CyberVision, Inc.
+ * Copyright 2016-2018 KaaIoT Technologies, LLC.
```

Comment:
I believe we should specify both companies in there. Do not change the previous line but add a new one with KaaIoT

Reply:
Done.